src/argilla/client/feedback/training/base.py

```diff
@@ -71,10 +71,20 @@ def __init__(
             gpu_id: the GPU ID to use when training a SpaCy model. Defaults to -1, which means that the CPU
                 will be used by default. GPU IDs start in 0, which stands for the default GPU in the system,
                 if available.
+            filter_by: A dict with key the field to filter by, and values the filters to apply. Currently only
+                defined for `response_status` filters. Can be one of: draft, pending, submitted, and discarded.
+                Defaults to `None` (no filter is applied).
             framework_kwargs: arguments for the framework's trainer. A special key (model_card_kwargs) is reserved
                 for the arguments that can be passed to the model card.
             **load_kwargs: arguments for the rg.load() function.
         """
+        if filter_by:
```

Comment:
I think we always need to pull RemoteFeedbackDatasets. Also could you include the sort_by as well?

Reply:
Included in [0b36807](https://github.com/argilla-io/argilla/pull/4120/commits/0b3680797b98d28e499551ec3b79073bfc365ffc)